mongodb-enterprise-openshift.yaml

```diff
@@ -106,20 +103,19 @@ rules:
   - list
   - watch
 
+
 ---
-kind: ClusterRoleBinding
```

Comment:
what's the reason for this change? CSR is a cluster resource, my understanding is that to grant permissions to it we need `ClusterRoleBinding`?